connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -2108,19 +2362,23 @@ ble_error_t Gap::startAdvertising(
 #if BLE_ROLE_BROADCASTER
 ble_error_t Gap::stopAdvertising(advertising_handle_t handle)
 {
+    tr_info("Advertising set %d: stop advertising", handle);
     ble_error_t status;
 
 #if BLE_FEATURE_EXTENDED_ADVERTISING
     if (handle >= getMaxAdvertisingSetNumber()) {
+        tr_error("Failed: handle cannot be greater than/equal to %d", getMaxAdvertisingSetNumber());
         return BLE_ERROR_INVALID_PARAM;
     }
 
     if (!_existing_sets.get(handle)) {
+        tr_error("Failed: non-existent advertising set");
         return BLE_ERROR_INVALID_PARAM;
     }
 #endif // BLE_FEATURE_EXTENDED_ADVERTISING
 
     if (!_active_sets.get(handle) || _pending_sets.get(handle)) {
+        tr_error("Failed: active advertising set");
```

Comment:
```suggestion
        tr_error("Failed: Cannot stop set - not active");
```

Reply:
Thanks. Not sure how I deduced that from the logic above.